examples/routing_service/monitoring/c++11/README.md

```diff
@@ -2,36 +2,29 @@
 
 ## Building the Example :wrench:
 
-To build this example, first run CMake to generate the corresponding build
-files. We recommend you use a separate directory to store all the generated
-files (e.g., ./build).
+In order to build this example, you need to define the variables `CONNEXTDDS_DIR`
+and `CONNEXTDDS_ARCH`. You can do so by exporting them manually, by sourcing
+the `rtisetenv` script for your architecture, or by passing them to the `cmake`
+command as arguments:
 
-```sh
+```bash
 mkdir build
 cd build
-cmake ..
-```
-
-Once you have run CMake, you will find a number of new files in your build
-directory (the list of generated files will depend on the specific CMake
-Generator). To build the example, run CMake as follows:
-
-```sh
+cmake -DCONNEXTDDS_DIR=<Connext DDS Directory> \     # If not exported
+      -DCONNEXTDDS_ARCH=<Connext DDS Architecture> \ # If not exported
+      -DBUILD_SHARED_LIBS=ON|OFF \
+      -DCMAKE_BUILD_TYPE=Debug|Release ..
 cmake --build .
 ```
 
-**Note**: if you are using a multi-configuration generator, such as Visual
-Studio solutions, you can specify the configuration mode to build as follows:
-
-```sh
-cmake --build . --config Release|Debug
-```
+**Note**: You do not need to define `CONNEXTDDS_ARCH` if you only have one
+architecture target installed in your system.
 
-Alternatively, you can use directly the generated infrastructure (e.g.,
-Makefiles or Visual Studio Solutions) to build the example. If you generated
-Makefiles in the configuration process, run make to build the example. Likewise,
-if you generated a Visual Studio solution, open the solution and follow the
-regular build process.
+Alternatively to using `cmake --build`, you can use directly the generated
```

Comment:
This is not important enough to go through another round of approvals, but this line would be better as, "Instead of using `cmake --build`, you can directly use the generated"

Reply:
but since I have another couple comments that are more important, might as well make this change, too